src/store/store.ts

```diff
@@ -0,0 +1,1218 @@
+      }
+      return this.getDefineCompanyStep.valid
+    },
+
+    /** Is true when the step is valid. */
+    isRestoreBusinessNameValid (): boolean {
+      return (
+        this.getBusinessNameValid &&
+        this.getNameTranslationsValid &&
+        this.getRestorationTypeValid &&
+        this.getApprovalTypeValid
+      )
+    },
+
+    /** Is true when the step is valid. */
+    isAddPeopleAndRolesValid (): boolean {
+      return this.getAddPeopleAndRoleStep.valid
+    },
+
+    /** Is true when the step is valid. */
+    isCreateShareStructureValid (): boolean {
+      return this.getCreateShareStructureStep.valid
+    },
+
+    /** Is true when the step is valid. */
+    isIncorporationAgreementValid (): boolean {
+      return this.getIncorporationAgreementStep.valid
+    },
+
+    /** Whether the subject filing is valid. */
+    isFilingValid (): boolean {
+      if (this.isIncorporationFiling) return this.isIncorporationApplicationValid
+      if (this.isDissolutionFiling) return this.isDissolutionValid
+      if (this.isRegistrationFiling) return this.isRegistrationValid
+      if (this.isRestorationFiling) return this.isRestorationValid
+      return false // should never happen
+    },
+
+    /** Whether all the dissolution steps are valid. */
+    isDissolutionValid (): boolean {
+      const isDocumentDeliveryValid = this.getDocumentDelivery.valid
+      const isCertifyValid = this.getCertifyState.valid && !!this.getCertifyState.certifiedBy
+
+      // only for Premium account
+      const isTransactionalFnValid = !this.isPremiumAccount || this.getTransactionalFolioNumberValid
+
+      // only for Staff role
+      const isCourtOrderValid = this.isRoleStaff ? this.getCourtOrderStep.valid : true
+      const isStaffPaymentValid = this.isRoleStaff ? this.getStaffPaymentStep.valid : true
+      const isCompletingPartyValid = this.isRoleStaff ? this.getCompletingParty?.valid : true
+
+      const isDissolutionDateValid = !!this.getDissolutionDate
+
+      const isEffectiveDateTimeValid = (this.isBaseCompany)
+        ? this.getEffectiveDateTime.valid
+        : true
+
+      if (this.isTypeFirm) {
+        return (
+          isDocumentDeliveryValid &&
+            isTransactionalFnValid &&
+            isCertifyValid &&
+            isCourtOrderValid &&
+            isStaffPaymentValid &&
+            isDissolutionDateValid &&
+            isCompletingPartyValid
+        )
+      }
+      return (
+        this.isDissolutionDefineDissolutionValid &&
+        this.isAffidavitValid &&
+        this.isResolutionValid &&
+        isDocumentDeliveryValid &&
+        isTransactionalFnValid &&
+        isCertifyValid &&
+        isEffectiveDateTimeValid &&
+        isCourtOrderValid &&
+        isStaffPaymentValid
+      )
+    },
+
+    /** Whether all the incorporation steps are valid. */
+    isIncorporationApplicationValid (): boolean {
+    // Base company steps
```

Comment:
Indent.

Try `npm run lint` and see if this get auto-fixed.

Reply:
`npm run lint` says No lint errors found! I will fix it manually

Reply:
Should be
```
    isIncorporationApplicationValid (): boolean {
      // Base company steps
      const isBaseStepsValid = (
```